spec/requests/api/v0/kafkadata_spec.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'ostruct'
+
+RSpec.describe Api::V0::EventsController::KafkaData do
+  let(:user_id) { 'd8388680-80cf-4fdb-95bb-829a46342115' }
+  let(:user_data) { { 'user_uuid': user_id } }
```

Comment:
unused

Reply:
Ah the user_data one - yup ,left over copy-pasta

Reply:
fixed